docs/ums/README.md

```diff
@@ -240,23 +373,36 @@ For this you need to create a policy with the following values:
 * **Subject**: All Users 
 * **Resource**: All Users
 
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
 // Select all active users  
 ums.select()  
  .where(field => field("active").isEqualTo(true))  
- .subscribe();  
+ .subscribe(user => {}, error=>{});   
 // Suspend Elon! 
 ums.update({ active: false })  
  .where(field => field("email").isEqualTo("Elon@tesla.com"))  
- .subscribe();  
+ .subscribe(user => {}, error=>{});    
 // Delete all suspended users  
 ums.delete()  
  .where(field => field("active").isEqualTo(false))  
- .subscribe(); 
+ .subscribe(user => {}, error=>{});   
 ```
+</template>
+<template v-slot:py>
+
+``` py
+  res = client.request(
+          method='GET',
+          url='/ums/users',
+          cond='[....]'
+        ) 
+  print(res)
```

Comment:
I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

Reply:
next release